### ISSUE TITLE
Create article variation for Poll story

### ISSUE THREAD

**Issue by phillipadsmith** [TheTyee/design-article.thetyee.ca#66]
Make an example page with an embedded Wufoo poll and a "The Barometer" header. Needs to be coordinated with Michelle to determine absolute "must have" requirements. 

Article:
http://thetyee.ca/Polls/2016/03/21/Fossil-Fuel-Investments/

**Comment by phillipadsmith** [comment]
@alexgreen Wondering your thought on accommodating the sidebar elements for this type of story? 

Essentially, what's required is:
- [ ] Visual indication of section (Barometer)
- [ ] Disclaimer ("Please note that Tyee Barometer polls are only intended as a quick and engaging non-scientific snapshot of our readers' opinions..."
- [ ] Option of included some text in the sidebar (currently the "Thanks Reader..." text)
- [ ] Option of included a Twitter widget (as seen at link above)

We'll also need to keep in mind what we're removing from narrow viewports, i.e., the disclaimer should stay.

Hmmm... now that I think that through, we may need to sit down with Michelle on this one and start to re-think it a bit.

Phillip.

**Comment by alexgreen** [comment]
Ok, I'll hold off until after the rethink.

My sense is that the main requirement that distinguishes this story from a regular story is:
There is no featured image (true?) yet we must accommodate a half-page ad. I can create a special aside to display the ad next to the story body when the view is wide enough.

> Disclaimer ("Please note that Tyee Barometer polls are only intended as a quick and engaging non-scientific snapshot of our readers' opinions..."

Could this go into a fact-box object? That way it will show up at the bottom of the story text in narrow views. Or maybe it could go into the (soon-to-be) new editor's note. Then it would always sit inline at the top of the story.

> Option of included a Twitter widget (as seen at link above)

This could be sized to fit into a normal aside. But why is there a twitter widget on Poll stories?

> Option of included some text in the sidebar (currently the "Thanks Reader..." text)

I think a fact-box or a messaging-box would be the logical choice to contain this.

**Comment by phillipadsmith** [comment]
> Ok, I'll hold off until after the rethink.

👍  Had that conversation with Michelle & Robyn, will summarize in Basecamp.

> There is no featured image (true?)

In fact, they're okay and in favour of having a featured image (possibly with a ribbon to indicate that it's a poll)

> yet we must accommodate a half-page ad.

No half-page is necessary, in fact. So I'm not sure how that would / would not work with the featured image? 

> Could this go into a fact-box object? That way it will show up at the bottom of the story text in narrow views. Or maybe it could go into the (soon-to-be) new editor's note. Then it would always sit inline at the top of the story.

Yes, those would be two good options indeed.

> This could be sized to fit into a normal aside. But why is there a twitter widget on Poll stories?

When they're sponsored, they've often put in the sponsor's Twitter feed. I'm wondering if it should just be an in-story figure like SoundCloud example? 

> I think a fact-box or a messaging-box would be the logical choice to contain this.

Good point.

Long story short on the conversation with Michelle and Robyn was: Poll stories will become just regular stories, and the existing elements (per your comments above) should be used to meet the requirements. 

However, the one item to be addressed is: How to show a sponsors' logo / branding on a sponsored poll? Is there an elegant way to do it that will also carry over to mobile views?

**Comment by phillipadsmith** [comment]
Hey there @MrBryan,

Just a note that I've prioritized the builder campaign prep this past Monday, and will need to focus on that again this Thursday morning -- so I'm running a few days behind on having every type of story template mocked-up for you with real content. 

Let me know if that's holding you back in any way. I'm aiming to have it all done for Thursday night (fingers crossed).

Phillip.

**Comment by MrBryan** [comment]
Hey Phillip no worries I have lots of little tasks to knock off and more planning before then

**Comment by alexgreen** [comment]
I have also been busy on other fronts over the few days. I should be back on this wednesday night or Thursday, vancouver time.

> No half-page is necessary, in fact.

Ok, I've removed that item from your above checklist of requirements.
But we do have to support bigbox ads in the positions (next to featured image and also within text column), correct?

> In fact, they're okay and in favour of having a featured image (possibly with a ribbon to indicate that it's a poll)

Does this mean they're also ok with having no featured image (never)? Or is a featured image a definite requirement? (sometimes? or always?)

> However, the one item to be addressed is: How to show a sponsors' logo / branding on a sponsored poll?

Can you provide a link to a sponsored poll?

**Comment by alexgreen** [comment]
To sum up (and subject to revision pending clarification re the featured image):

**Poll Story brief**:
Poll stories will become just regular stories, and the existing elements should be used to meet the requirements.

Requirements:
- [ ] **Visual indication of section** (Barometer)
- [ ] **Disclaimer** ("Please note that Tyee Barometer polls are only intended as a quick and engaging non-scientific snapshot of our readers' opinions...") _This could go into a fact-box object. That way it will show up at the bottom of the story text in narrow views. Or it could go into the (soon-to-be) new editor's note. Then it would always sit inline at the top of the story._
- [ ] **Option of included some text in the sidebar** (currently the "Thanks Reader..." text) _I think a fact-box or a messaging-box would be the logical choice to contain this._
- [ ] **Option of including a Twitter widget** (as seen at link above) _When they're sponsored, they've often put in the sponsor's Twitter feed. This could be sized to fit into a normal aside. I'm wondering if it should just be an in-story figure like SoundCloud example?_
- [ ] **Featured Image?** always?. _In fact, they're okay and in favour of having a featured image (possibly with a ribbon to indicate that it's a poll)_
- [ ] **Bigbox Ad** in the usual positions (next to featured image and also in main column)

**Comment by phillipadsmith** [comment]
> To sum up (and subject to revision pending clarification re the featured image):

Correct.

> Visual indication of section (Barometer)

Yes, but this could be a ribbon on a feature photo.

> Disclaimer ("Please note that Tyee Barometer polls are only intended as a quick and engaging non-scientific snapshot of our readers' opinions...") This could go into a fact-box object. That way it will show up at the bottom of the story text in narrow views. Or it could go into the (soon-to-be) new editor's note. Then it would always sit inline at the top of the story.

Fact box please. :-)

> Option of included some text in the sidebar (currently the "Thanks Reader..." text) I think a fact-box or a messaging-box would be the logical choice to contain this.

Agreed.

>  Option of including a Twitter widget (as seen at link above) When they're sponsored, they've often put in the sponsor's Twitter feed. This could be sized to fit into a normal aside. I'm wondering if it should just be an in-story figure like SoundCloud example?

I think I'm leaning toward that idea too, to be honest. Hard to see how it would look good in the sidebar.

>  Featured Image? always?. In fact, they're okay and in favour of having a featured image (possibly with a ribbon to indicate that it's a poll)

Always if we say so, yes. They're fine with always choosing an image.

> Bigbox Ad in the usual positions (next to featured image and also in main column)

Could be big box or half-page. Currently, the size of "sponsor branding" is half page on poll stories. The challenge is that these are not currently "ads" per se because they're not delivered by the ad server. However, that's a back-end issue, not a design issue. 

Let me know if that confirms the requirements?

**Comment by alexgreen** [comment]
> Let me know if that confirms the requirements?

Just a couple clarifications:

> Could be big box or half-page. Currently, the size of "sponsor branding" is half page on poll stories.

So we do need to support a half-page ad (or "sponsor branding") placement? Or we don't? I can't tell if this is a requirement or not. Are we free to make an executive decision on this?

> Featured Image? always?. In fact, they're okay and in favour of having a featured image (possibly with a ribbon to indicate that it's a poll)
> Always if we say so, yes.

Does this mean they're also ok with never having a featured image on poll stories? Omitting the featured image could be part of how we differentiate Poll stories. (... although we can't omit the featured image if there needs to be support for a half-page ad/sponsor placement.)

> The challenge is that these are not currently "ads" per se because they're not delivered by the ad server.

So Poll stories will have "sponsor branding" (which is the same size as ad blocks) and _**also**_ ads? Or does the sponsor branding replaces the ads (i.e. there's never any ads in these stories)?

**Comment by phillipadsmith** [comment]
> So we do need to support a half-page ad (or "sponsor branding") placement? Or we don't? I can't tell if this is a requirement or not. Are we free to make an executive decision on this?

Yes, we need to support "sponsor branding" placement (when the poll is sponsored). 

> Does this mean they're also ok with never having a featured image on poll stories? Omitting the featured image could be part of how we differentiate Poll stories. (... although we can't omit the featured image if there needs to be support for a half-page ad/sponsor placement.)

Yes, we're okay to omit, but as you point out that doesn't play nicely with the half-page ad placement. So I think it's easier to have it.

> So Poll stories will have "sponsor branding" (which is the same size as ad blocks) and also ads? Or does the sponsor branding replaces the ads (i.e. there's never any ads in these stories)?

No ads, only sponsor branding.

**Comment by phillipadsmith** [comment]
> Yes, we need to support "sponsor branding" placement (when the poll is sponsored).

Polls are not sponsored all the time, however, so the poll needs to be able to survive without a half-page advertisement also (or with a house half-page advertisement in place of the sponsor's).

**Comment by alexgreen** [comment]
**Poll Story brief**:
Poll stories will become just regular stories, and the existing elements should be used to meet the requirements.

Requirements pending further updates:
- [ ] a **"The Barometer" header**
- [ ] an **embedded Wufoo poll**
- [ ] **Visual indication of section** (Barometer)
- [ ] **Disclaimer** ("Please note that Tyee Barometer polls are only intended as a quick and engaging non-scientific snapshot of our readers' opinions...") _This should go into a fact-box object. That way it will show up at the bottom of the story text in narrow views._
- [ ] **Option of including some text in the sidebar** (currently the "Thanks Reader..." text) _I think a fact-box or a messaging-box would be the logical choice to contain this._
- [ ] **Option of including a Twitter widget** (as seen at link above) _When they're sponsored, they've often put in the sponsor's Twitter feed. Place this as an in-story figure (like soundcloud). (An alternative is to size it to fit into a normal aside._
- [ ] **Featured Image** ALWAYS. We need a featured image so that we can easily place the half-page "sponsor branding" ad block.
- [ ] **Half-Page ad** -- for "sponsor branding"

**There are no ads on these pages, only sponsor branding.** Polls are not sponsored all the time, however, so the poll needs to be able to survive without a half-page advertisement also (or with a house half-page advertisement in place of the sponsor's).

[See also Style Poll Stories #150]

**Comment by alexgreen** [comment]
See also for further specs: https://basecamp.com/1929296/projects/3301064/messages/60624317#comment_446644553
